docs/src/main/asciidoc/doc-contribute-docs-howto.adoc

```diff
@@ -69,6 +69,68 @@ There must also be a line break before and after the abstract.
 
 For more information about the minimum header requirements, see xref:{doc-guides}/doc-reference.adoc#document-structure[Document structure] on the "Quarkus style and content guidelines" page.
 
+[[anchors-howto]]
+=== Cross-referencing in-file and cross-file content by using anchors
```

Comment:
Just a very minor suggestion: Use the imperative verb form instead of the gerund form, as per the upstream Quarkus style preference for titles. 

```suggestion
=== Cross-reference in-file and cross-file content by using anchors
```

Reply:
Done and done